### PR TITLE
Frontend/create login and register place

### PR DESCRIPTION
**Pull request by katarzynagodlewska**

### CODE REVIEW

File: frontend/src/components/Login.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    password: "",
+  };
+
+  @user.State
+  public email!: string;
+
+  @user.Action
+  public login!: (userData: UserLoginModel) => void;
+
+  public submitForm(): void {
```
Comment (KasperCisewski) [inline frontend/src/components/Login.vue:39]: Why here function has named : submitForm() but on REgister.Vue component function has handleSubmit() name?

File: frontend/src/styles/_normalize.scss [hunk 2]
```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
```
Comment (KasperCisewski) [inline frontend/src/styles/_normalize.scss:3]: we need thats comments in this file ?

Comment (katarzynagodlewska) [reply frontend/src/styles/_normalize.scss:3]: I think yes

File: frontend/src/components/Login.vue (hunk 1 above)
Comment (katarzynagodlewska) [reply frontend/src/components/Login.vue:39]: Changed

**Pull request merged**